src/marten/conf/global_settings.cr

```diff
@@ -198,6 +210,7 @@ module Marten
         @middleware = Array(Marten::Middleware.class).new
         @port = 8000
         @port_reuse = true
+        @referrer_policy = "strict-origin-when-cross-origin"
```

Comment:
I know that `strict-origin-when-cross-origin` is usually the browser default. That said, I am under the impression that the framework should push for a safer value by default (like `same-origin`). WDYT?

Reply:
Yes, totally agree!